plugins/vault-backend/README.md

```diff
@@ -8,6 +8,20 @@ Vault is an identity-based secrets and encryption management system. A secret is
 
 This plugins allows you to view all the available secrets at a certain location, and redirect you to the official UI so backstage can rely on LIST permissions, which is safer.
 
+## New Backend System
+
+The Vault backend plugin has support for the [new backend system](https://backstage.io/docs/backend-system/), here's how you can set that up:
+
+In your `packages/backend/src/index.ts` make the following changes:
+
+```diff
+  import { createBackend } from '@backstage/backend-defaults';
+  const backend = createBackend();
+  // ... other feature additions
+  backend.add(import('@backstage/plugin-vault-backend'));
+  backend.start();
+```
```

Comment:
```suggestion
```diff
  import { createBackend } from '@backstage/backend-defaults';
  const backend = createBackend();
  // ... other feature additions
+ backend.add(import('@backstage/plugin-vault-backend'));
  backend.start();
```
```

Reply:
sorry yeah it wasn't so clear 🙈 I've just added a `+` on line 21 to mark the changes that adopters need to be done 😅

Reply:
Hi [vinzscam](https://github.com/vinzscam)
 I have tried to address the comment, if there are any other please let me know.

Reply:
Hi [vinzscam](https://github.com/vinzscam)
Are there any updates for this PR

Thanks